nilearn/regions/rena_clustering.py

```diff
@@ -346,9 +346,9 @@ def nearest_neighbor_grouping(X, connectivity, n_clusters, threshold=1e-7):
     return reduced_connectivity, reduced_X, labels
 
 
-def recursive_neighbor_agglomeration(X, mask_img, n_clusters,
-                                     n_iter=10, threshold=1e-7,
-                                     verbose=0):
+def _recursive_neighbor_agglomeration(X, mask_img, n_clusters,
```

Comment:
I think that this is a high-level function that can be called/used outside of this module. I would not make it private. But I guess that this is open for discussion